src/pages/edge/identity-for-edge-network/faq.md

```diff
@@ -113,6 +113,17 @@ To clear the identifiers used by the Adobe Solutions extensions, call [setPrivac
 
 To clear the identifiers used by the Edge extensions, call [resetIdentities](../../home/base/mobile-core/api-reference.md#resetidentities)
 
+## Q: In what cases is the Experience Cloud ID (ECID) reset or cleared?
+
+A: The ECID is reset or cleared in the following cases:
+
+* Cleared when the app is uninstalled.
+* Reset using the [`resetIdentities`](../../home/base/mobile-core/api-reference.md#resetidentities) API.
+* Reset when the app's local persistence cleared (by the app developer, other SDKs, etc.).
```

Comment:
Rephrasing suggestion:
Reset in the case of app local persistence is cleared for any reason in the app implementation. To learn more, see where the Mobile SDK store identities data. (relative link to https://developer.adobe.com/client-sdks/resources/faq/#where-does-the-sdk-store-identities-and-preferences-on-the-app) 

And I think we can remove the details about persistence.

Reply:
yes, that looks good to me - cc @praveek